go.mod

```diff
@@ -3,9 +3,7 @@ module github.com/mattermost/mattermost-plugin-demo
 go 1.19
 
 require (
-	github.com/mattermost/mattermost-plugin-api v0.1.4
-	// mmgoget: github.com/mattermost/mattermost-server/v6@v7.9.4 is replaced by -> github.com/mattermost/mattermost-server/v6@ea3f5187c3
-	github.com/mattermost/mattermost-server/v6 v6.0.0-20221012175353-8cb6718a9bcc
+	github.com/mattermost/mattermost/server/public v0.0.8-0.20230808080038-0753def44509
```

Comment:
If we need something newer than `v0.0.7`, can we tag and depend on that here vs. relying on a commit hash dependency?

Reply:
I'm currently working on getting https://github.com/mattermost/mattermost/pull/24009 to a state where it can be reviewed (and hopefully merged). Once the code is in `master`, I'm happy to tag another release.

0/5 if we should wait to merge this PR until a new tag is released.

Reply:
https://pkg.go.dev/github.com/mattermost/mattermost/server/public@v0.0.8 has been published.